frontend/.env

```diff
@@ -0,0 +1,3 @@
+# The backend API for chat endpoint.
+NEXT_PUBLIC_CHAT_API=http://localhost:8000/api/chat
```

Comment:
@marlenezw Is the .env intended to be checked in?

Reply:
Ah, those env files were auto-generated by Llama. I'll delete them